qiskit_optimization/converters/maximize_to_minimize.py

```diff
@@ -0,0 +1,78 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Converter to convert a maximization problem to minimization problem."""
+
+import copy
+import logging
+from typing import Optional, Union, Tuple, List
+
+import numpy as np
+
+from .quadratic_program_converter import QuadraticProgramConverter
+from ..exceptions import QiskitOptimizationError
+from ..problems.quadratic_objective import QuadraticObjective
+from ..problems.quadratic_program import QuadraticProgram
+
+logger = logging.getLogger(__name__)
```

Comment:
Minor comment: as there is no logging done in this file we can remove this from the file since its not used.

Reply:
Removed the logger